components/src/lists/TitledList.tsx

```diff
@@ -1,53 +1,53 @@
-// @flow
 // TitledList component
 import * as React from 'react'
 import cx from 'classnames'
 
 import styles from './lists.css'
-import { type IconName, Icon } from '../icons'
+import { Icon } from '../icons'
+import type { IconName } from '../icons'
 
-export type TitledListProps = {|
+export interface TitledListProps {
   /** text of title */
-  title: string,
+  title: string
   /** optional icon left of the title */
-  iconName?: ?IconName,
+  iconName?: IconName | null
```

Comment:
Technically missing `| undefined`

```suggestion
  iconName?: IconName | null | undefined
```